CHANGELOG.md

```diff
@@ -4,6 +4,17 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog][keepachangelog] and this project adheres to [Semantic Versioning][semver].
 
+## UNRELEASED
+
+### Added
+
+- Minimal required PHP version now is `8.0`
+- Minimal required dependency `illuminate/support` now is `~9.0`
+
```

Comment:
Laravel 9 requires typed Collection<TKey, TValue> in their signature, Laravel 8 doesn't.

Phpstan throws error in that cases: 

- If we remove Collection key and value typing, it doesn't match with Laravel 9 signature.
- If we set Collection<TKey, TValue>, it doesn't match with Laravel 8 signature.


That's why we boosted minimal required Laravel version to 9.0